Fastfile

```diff
@@ -573,66 +665,7 @@ platform :ios do
     )
   end
 
-  lane :check_store_env_variables do |options|
-    variables = shared_env_variables + [
-      "APP_ID", 
-      "SCHEME",
-      "IPA_NAME",
-      "GOOGLE_SERVICE_INFO_PLIST_PATH",
-      "APP_STORE_BUILD_CONFIGURATION",
-      "FASTLANE_ITC_TEAM_NAME",
-    ]
-
-    ensure_env_vars(
-      env_vars: variables
-    )
-  end
-
-  lane :check_prod_env_variables do |options|
-    variables = shared_env_variables + [
-      "APP_ID", 
-      "PROD_SCHEME",
-      "PROD_IPA_NAME",
-      "PROD_TRYOUTS_APP_ID",
-      "PROD_TRYOUTS_API_TOKEN",
-      "PROD_GOOGLE_SERVICE_INFO_PLIST_PATH",
-      "ADHOC_BUILD_CONFIGURATION"
-    ]
-
-    ensure_env_vars(
-      env_vars: variables
-    )
-  end
-
-  lane :check_staging_env_variables do |options|
-    variables = shared_env_variables + [
-      "STAGING_APP_ID", 
-      "STAGING_SCHEME",
-      "STAGING_IPA_NAME",
-      "STAGING_TRYOUTS_APP_ID",
-      "STAGING_TRYOUTS_API_TOKEN",
-      "STAGING_GOOGLE_SERVICE_INFO_PLIST_PATH",
-      "ADHOC_BUILD_CONFIGURATION"
-    ]
-
-    ensure_env_vars(
-      env_vars: variables
-    )
-  end
-
-  lane :check_preprod_env_variables do |options|
-    variables = shared_env_variables + [
-      "PREPROD_APP_ID", 
-      "PREPROD_SCHEME",
-      "PREPROD_IPA_NAME",
-      "PREPROD_TRYOUTS_APP_ID",
-      "PREPROD_TRYOUTS_API_TOKEN",
-      "PREPROD_GOOGLE_SERVICE_INFO_PLIST_PATH",
-      "ADHOC_BUILD_CONFIGURATION"
-    ]
-
-    ensure_env_vars(
-      env_vars: variables
-    )
+  lane :env_value do |options|
+    ENV[options[:key]]
```

Comment:
I think we can remove this method as well, it seems it doesn't do anything very useful. Instead of `ENV[env_variables[:core][:slack_webhook_url]]`, you will write `env_value(key: env_variables[:core][:slack_webhook_url])`.

Reply:
👍